vega_sim/api/trading.py

```diff
@@ -292,7 +292,7 @@ def submit_simple_liquidity(
         delta_sell:
             int, the offset from reference point for the sell side of LP
     """
-    print(f"Submitting with buy {delta_buy} and sell {delta_sell}")
+    # print(f"Submitting liquidity with buy {delta_buy} and sell {delta_sell}")
```

Comment:
Let's try to avoid "commented out code" in the main branch.

Reply:
I dislike that I even added a print at all. Kill it please @Galen-Cao :)

Reply:
No problem.